### ISSUE TITLE
Metrics drop during new training (after completion of initial training)

### ISSUE THREAD

**Issue by dexforint** [ultralytics/ultralytics#13891]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

After I trained Yolov8s model (all epochs I needed), I wanted to finetune my model more. So I launched training process with params:

```python
model = YOLO(f'./last.pt')
model.to("cuda");

model.train(
    data = '/kaggle/input/ldt23-data-yaml/data.yaml',
    epochs = 70,
    batch = -1,
    workers = 4,
    deterministic=False,
    label_smoothing=0.05,
    project='ldt23', 
    name=f"yolov8{MODEL_SIZE}",
    imgsz=640,
#     resume=True, # I commented this line because this is first run in second training (after first 70 epochs)
)

```

Metrics droped in the new training.

Please help me to understand how to continue training without drop in metrics. What params should I set and what values should I enter?

### Additional

![opera_idKV53YyVI](https://github.com/ultralytics/ultralytics/assets/22598276/54efe18c-9f10-481b-ad18-8efe96000966)

**Comment by glenn-jocher** [comment]
@dexforint hi there! 👋

Thank you for providing the details of your training process. To help you effectively continue training without a drop in metrics, let's consider a few key points:

1. **Learning Rate Adjustment**: When fine-tuning a model, it's often beneficial to reduce the learning rate. The initial learning rate used for training from scratch might be too high for fine-tuning, leading to a drop in metrics. You can adjust the learning rate using the `lr0` parameter:
    ```python
    model.train(
        data='/kaggle/input/ldt23-data-yaml/data.yaml',
        epochs=70,
        batch=-1,
        workers=4,
        deterministic=False,
        label_smoothing=0.05,
        project='ldt23', 
        name=f"yolov8{MODEL_SIZE}",
        imgsz=640,
        lr0=0.001  # Adjust this value as needed
    )
    ```

2. **Warmup Period**: Introducing a warmup period can help stabilize training when resuming or fine-tuning. This gradually increases the learning rate from a low value to the initial learning rate over a few epochs:
    ```python
    model.train(
        data='/kaggle/input/ldt23-data-yaml/data.yaml',
        epochs=70,
        batch=-1,
        workers=4,
        deterministic=False,
        label_smoothing=0.05,
        project='ldt23', 
        name=f"yolov8{MODEL_SIZE}",
        imgsz=640,
        lr0=0.001,  # Adjust this value as needed
        warmup_epochs=3  # Number of warmup epochs
    )
    ```

3. **Check for Overfitting**: Ensure that your model is not overfitting. Monitor the training and validation loss. If the validation loss starts increasing while the training loss decreases, it might indicate overfitting. You can use techniques like early stopping or regularization to mitigate this.

4. **Data Augmentation**: Ensure that your data augmentation strategies are consistent between the initial training and fine-tuning phases. Inconsistent augmentation can lead to a drop in metrics.

5. **Hyperparameter Tuning**: Fine-tuning might require different hyperparameters than the initial training. Experiment with different values for parameters like `momentum`, `weight_decay`, and `label_smoothing`.

6. **Resume Training**: If you intend to continue training from where you left off, ensure that the `resume` parameter is set correctly. However, since you mentioned this is the first run in the second training, this might not be necessary.

If the issue persists, please ensure you are using the latest versions of `torch` and `ultralytics`:
```bash
pip install --upgrade torch ultralytics
```

Feel free to share any additional details or logs if the problem continues. We're here to help! 😊
